.github/workflows/build.yml

```diff
@@ -0,0 +1,70 @@
+name: 'Build'
+
+on: [push]
+
+jobs:
+  release-content:
+    name: 'Create GitHub Release'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Run read-yaml action
+        id: yaml-data
+        uses: jbutcher5/read-yaml@main
+        with:
+          file: './plugin.yaml'
+          key-path: '["version"]'
+
+      - name: Display read-yaml output
+        run: echo "${{ steps.yaml-data.outputs.data }}"
+      - uses: mukunku/tag-exists-action@v1.2.0
```

Comment:
do we really need this check?
if we broke `tags` manually we have to fix it manually too, isn't it?
or do i miss any usecases?